src/main/java/bio/terra/service/snapshotbuilder/utils/SearchConceptsQueryBuilder.java

```diff
@@ -1,6 +1,9 @@
 package bio.terra.service.snapshotbuilder.utils;
 
+import static bio.terra.service.snapshotbuilder.utils.CriteriaQueryBuilder.getOccurrenceTableFromDomain;
```

Comment:
In general I think we should avoid `static` imports outside of specific cases in our test code. A `static` import like this makes it harder to see where a method reference is coming from.

Reply:
Removed!